tailcfg/derpmap.go

```diff
@@ -29,6 +34,11 @@ func (m *DERPMap) RegionIDs() []int {
 	return ret
 }
 
+// DERPHomeHints contains hints from the server about which DERP servers to use as the home server.
+type DERPHomeHints struct {
+	RegionWeight map[int]float64 // Region ID to its latency scaling factor
```

Comment:
specify what nil means. Probably it should mean unchanged, but an empty non-nil maps means clear all scaling factors (reset everything back to 1)

And specify that if a region doesn't have a scaling factor in the map, that means 1?